domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -4348,6 +4349,10 @@ public void startPostgresTester() {
     }
     postgreSQLClientConfig.put(PORT, postgresTester.getPort());
     postgreSQLClientConfig.put(HOST, postgresTester.getHost());
+
+    // TODO If we enable these 8 tests fail.
+    postgreSQLClientConfig.put(HOST_READER, postgresTester.getReadHost());
+    postgreSQLClientConfig.put(PORT_READER, postgresTester.getReadPort());
```

Comment:
@julianladisch Two tests fail (not 8 as the comment states). These failed tests are `PostgresClientIT.getNewInstanceSharedPool` and `GeneralFunctionsIT.upsert`. Any insight as to what could be happening here?

Reply:
@julianladisch I fixed both of the tests that were failing. For the upsert test fix, if I get the connection from `getSQLReadConnection` rather than relying `withReadConnection` it succeeds. The only difference I can see is that one news up a new `Conn` object.